src/main/java/com/google/research/bleth/servlets/NewExperimentSimulationServlet.java

```diff
@@ -0,0 +1,124 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.research.bleth.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.EntityNotFoundException;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.appengine.api.datastore.Transaction;
+import com.google.appengine.api.datastore.TransactionOptions;
+import com.google.research.bleth.simulator.AbstractSimulation;
+import com.google.research.bleth.simulator.AwakenessStrategyFactory;
+import com.google.research.bleth.simulator.MovementStrategyFactory;
+import com.google.research.bleth.simulator.Schema;
+import com.google.research.bleth.simulator.StrategiesMapper;
+import com.google.research.bleth.simulator.TracingSimulation;
+
+import java.io.IOException;
+import java.util.logging.Logger;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * A servlet used for creating and running a new simulation associated to an experiment,
+ * and updating experimentsToSimulations database entity (which maps experiments to their associated simulations).
+ */
+@WebServlet("/new-experiment-simulation")
+public class NewExperimentSimulationServlet extends HttpServlet {
+    private static final DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    private static final Logger log = Logger.getLogger(NewExperimentSimulationServlet.class.getName());
+
+    @Override
+    public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+        log.info("Experiment Simulation Worker Servlet invoked.");
```

Comment:
Why do you need this log line?
You can search for this servlet in the logs by just querying for `protoPayload.resource="/new-experiment-simulation"`.